locations/fblipke.yml

```diff
@@ -0,0 +1,75 @@
+---
+location: fblipke
+location_nice: FBLipke - Freifunk Berlin
+latitude: 52.484948320
+longitude: 13.443380903
+altitude: 42
+height: 13
+contact_name: 'Guido L.'
+contact_nickname: 'LipkeGu'
+contacts:
+  - 'contact@fblipke.de'
+hosts:
+  - hostname: fblipke-core
+    role: corerouter
+    model: "x86_64"
+    wireless_profile: freifunk_default
+    low_flash: true
+  - hostname: fblipke-ap01
+    role: ap
+    model: "ubnt_nanostation_m2_xw"
+    wireless_profile: freifunk_default
+    low_flash: true
+
+ipv6_prefix: "2001:bf7:850:1500::/56"
+
+networks:
+  - vid: 20
+    role: mesh
+    name: 11s_n_2g
+    prefix: 10.248.2.200/32
+    ipv6_subprefix: -20
+    mesh_ap: fblipke-core
+    mesh_radio: 11g_standard
+    mesh_iface: mesh
+  - vid: 40
+    role: dhcp
+    prefix: 10.248.3.128/27
+    ipv6_subprefix: 0
+    inbound_filtering: true
+    enforce_client_isolation: true
+    no_corerouter_dns_record: true
+    assignments:
+      fblipke-core: 129
+      fblipke-ap01: 130
+  - vid: 42
+    role: mgmt
+    prefix: 10.248.3.160/28
+    gateway: 1
+    ntp: 1
+    ipv6_subprefix: 1
+    assignments:
+      fblipke-core: 1
+      fblipke-ap01: 2
+  - vid: 50
+    role: uplink
+
+  - role: tunnel
+    ifname: ts_wg0
+    mtu: 1280
+    prefix: 10.248.3.177/32
+    wireguard_port: 51820
+
+location__wireless_profiles__to_merge:
+  - name: foobar
+    devices:
+      - radio: 11g_standard
+        legacy_rates: false
+        country: DE
+    ifaces:
+      - mode: ap
+        ssid: berlin.freifunk.net
+        encryption: none
+        network: dhcp
+        radio: [11a_standard, 11g_standard]
+        ifname_hint: ff
```

Comment:
please remove this. Your configuration stated in the hosts section that it uses the default profile and therefore you don't need it. You only need to override the wireless_profiles if you want the APs to announce networks that are not the default. Have a look at w38b for a complex example.